src/server.py

```diff
@@ -30,10 +31,31 @@ async def handler(websocket, path):
         print(f"Connection error: {e}")
 
 
-async def main():
+async def start_websocket_server():
     async with websockets.serve(handler, "0.0.0.0", 8000):
         await asyncio.Future()  # run forever
 
 
+async def handle_health_check(request):
+    return web.Response(text="Server is healthy")
```

Comment:
I would return just `"OK"`. Not sure what the backend is returning, but probably we could agree on this.